articles/third-party/third-sd-hystax.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Hystax Acura Cloud Migration from UKCloud Service Definition | UKCloud Ltd
+description: Provides an overview of what is provided by the Hystax Acura Cloud Migration from UKCloud service
+services: third-party
+author: Sue Highmoor
+toc_rootlink: Hystax Acura Cloud Migration
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Hystax Acura Cloud Migration from UKCloud Service Definition
+toc_fullpath: Hystax Acura Cloud Migration/third-sd-hystax.md
+toc_mdlink: third-sd-hystax.md
+---
+
+# Portworx Enterprise from UKCloud Service Definition
+
+## Why UKCloud?
+
+UKCloud is dedicated to helping the UK Public Sector and UK citizens by delivering more choice and flexibility through safe and trusted cloud technology. We own and operate a UK-sovereign, industry-leading, multi-cloud platform, located within the Government’s Crown Campus, offering multiple cloud technologies, including VMware, Azure, OpenStack, OpenShift and Oracle. This enables customers to choose the right technology for creating new workloads or migrating existing applications to the cloud.
+
+We recognise the importance of government services in making the country run smoothly, which is why we include the highest level of support to all our customers at no extra cost. This includes a dedicated 24/7 UK telephone and ticket support, and Network Operations Centre (NOC) utilising protective and proactive monitoring tools, and access to UKCloud’s technical experts.
+
+![UKCloud services](images/ukc-services.png)
+
+## What is Hystax Acura Cloud Migration from UKCloud?
+
```

Comment:
Then remove as this will go on the live site.

Reply:
Removed entire service definition as it was added in error